### PR TITLE
Update sort-keys-recursive to version 2.0.0

### PR DESCRIPTION
**Pull request by matteocng**
Commits:
- docs: fix typos and rearrange some sentences
- feat: update sort-keys-recursive to version 2.0.0

Fixes #35.

### CODE REVIEW

File: lib/Finepack.coffee [hunk 1]
```diff
@@ -45,10 +46,10 @@ module.exports = (data, opts = {}, cb) ->
...
   return report.requiredMessage(cb, input) if validation.required
 
-  # The default sort is alphabetically
-  # after that we move some esthetic fields positions.
-  input = sort input
+  # The default sort order is alphabetical.
+  input = sort input, opts.sortOptions
 
+  # After sorting, we move some fields in new positions for esthetic effect.
   for key in KEYWORDS.sort when input[key]?
```
Comment (Kikobeats) [inline lib/Finepack.coffee]: Do you think we can avoid this piece of code? I mean, same behavior, but using `sort-keys-recursive` benefits.

**Comment by Kikobeats** [comment]
Thanks for that. I added comments 😄

File: lib/Finepack.coffee (hunk 1 above)
Comment (matteocng) [reply lib/Finepack.coffee]: Do you mean refactoring the code that sorts by `KEYWORDS.sort` or just removing that comment?

Comment (Kikobeats) [reply lib/Finepack.coffee]: the first, but not necessary, now works.

**Comment by Kikobeats** [comment]
Can I wait for something more or time to merge? 😄

**Comment by matteocng** [comment]
Sorting a second time using `sort-keys-recursive` and passing `compareFunction` works but it doesn't seem ideal, because the function would be applied to all keys recursively, not just the keys at the first level.

What do you think about this instead:
```coffeescript
# After sorting, we move some fields in new positions for esthetic effect.
compareFunction = (a, b) -> KEYWORDS.sort.indexOf(a) - KEYWORDS.sort.indexOf(b)
output = Object.keys(input).sort(compareFunction).reduce (acc, key) ->
 acc[key] = input[key]
 acc
, {} # Object, second argument of reduce(), initial value of 'acc'.
```

**Comment by Kikobeats** [comment]
@matteocng  I like this new approach, can you commit it? want to know if tests are green with this change

**Comment by matteocng** [comment]
I have updated the commit, also adding a small clarification about the behaviour of the default sort function (and link to MDN) in the `README` and code comments.

**Pull request merged**